### ISSUE TITLE
Prepare for release

### ISSUE THREAD

**Issue by tsurdilo** [serverlessworkflow/specification#122]
We are ready to do another release. Please add here any issues/concerns. 

The release need to happen for the SDKs as well and also be reflected on website.

**Comment by ricardozanini** [comment]
@tsurdilo do you have a date in mind? I'll take some time off to work on the SDK then, so we could have more to offer in this new version.

**Comment by tsurdilo** [comment]
@ricardozanini was hoping for end of the month if possible. I think for release its enough just to update the Go SDK to the latest version of the json schema. WDYT?

**Comment by ricardozanini** [comment]
I think it's fine! I'll work on the SDK at the end of this week. We should be fine for the release!

**Comment by tsurdilo** [comment]
I'd like to propose the release candidate: 1.0.0-rc.1
What do you guys think?

**Comment by manuelstein** [comment]
It feels awkward going from 0.1 straight to 1.0. I'm trying to understand why the bump. If it's for marketing, why would now be a good time? Could maybe KubeCon NA on 17 Nov be a good target? We could bump to 0.2 now and assemble a major release until 9 Nov - our last call just before KubeCon. Until then we have two more calls in Sep and two more in Oct according to our current schedule.

**Comment by manuelstein** [comment]
On today's community call we discussed options.
Of course, spec is different from software releases.

Generally, version 1.0 often marks the first "complete" set of features that provides a usable version.

Major release numbers are typically bumped when a lot of changes have led to a version that does not provide backwards compatibility.

We have indeed changed enough to say that the major version number should be increased (0->1)
Also, the current status provides a usable set of features, so it can be considered complete enough and out of the draft status.
So, skipping 0.2 and going straight to 1.0 sounds reasonable.

The timing would be best some time before kubecon na, and at best we'd want to use as many channels as possible to announce the 1.0 release, so we can stage release candidates with feature freeze and sufficient time for bugfixing to finish the release some time in October or before Nov 17 latest.

So now, we need a schedule aligned with our community calls :-) any suggestions?
